composer/7/Dockerfile

```diff
@@ -23,6 +23,6 @@ VOLUME ["/app"]
 
 # Define working directory
 WORKDIR /app
-
+ENTRYPOINT ["composer"]
 # Set up the command arguments
 CMD ["composer"]
```

Comment:
this is why, previously I asked
https://github.com/phalcon/dockerfiles/pull/10#discussion_r106063750